app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class='text-center mt-5'><%= @recipe.name %></h1>
+
+<section class="text-center mt-5">
+  <%= render @recipe %>
+  <div class="container my-5 ">
+    <div class="row">
+      <div class="d-flex justify-content-end align-content-center gap-5">
+        <%= button_to "Generate Shopping List", generate_shopping_list_recipe_path(@recipe), method: :post, class: "btn btn-primary" %>
+
+        <% if current_user == @recipe.user %>
+          <%= link_to "Add Ingredient", new_recipe_recipe_food_path(@recipe), class: "btn btn-success" %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+  <div class='container'>
```

Comment:
- [ ] The button public that will allow the user to make a recipe public is missing from you design, Kindly add it to follow the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)

Reply:
Hi @LionRouge1, have you tried running the app, because the button public is availabe of the recepes details page